backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/api/InvoiceCandidateQuery.java

```diff
@@ -34,6 +34,9 @@ public class InvoiceCandidateQuery
 	String headerAggregationKey;
 	BPartnerId salesRepBPartnerId;
 	InstantInterval dateOrderedInterval;
+	DocTypeId orderDocTypeId;
+	String orderDocumentNo;
+	List<String> orderLines;
```

Comment:
pls annotate all fields with non null / nullable
instead of List<String> orderLines pls use ImmutableList or even better ImmutableSet